content/en/docs/instrumentation/java/manual.md

```diff
@@ -704,18 +704,47 @@ instrument.
 
 ### Auto Configuration
 
-To configure the OpenTelemetry SDK based on the standard set of environment
-variables and system properties, you can use the
-`opentelemetry-sdk-extension-autoconfigure` module.
+Instead of manually creating the `OpenTelemetry` instance by using the SDK builders
+directly from your code, it is also possible to use the SDK auto-configuration extension 
+through the `opentelemetry-sdk-extension-autoconfigure` module.
+
+This module is made available by adding the following dependency to your application.
+
+```xml
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-jaeger</artifactId>
+</dependency>
+```
+
+It allows you to auto-configure the OpenTelemetry SDK based on a standard set of supported 
+environment variables and system properties.
+Each environment variable has a corresponding system property named the same way but as 
+lower case and using the `.` (dot) character instead of the `_` (underscore) as separator.
+
+The logical service name can be specified via the `OTEL_SERVICE_NAME` environment variable 
+(or `otel.service.name` system property).
+
+The traces, metrics or logs exporters can be set via the `OTEL_TRACES_EXPORTER`, 
+`OTEL_METRICS_EXPORTER` and `OTEL_LOGS_EXPORTER` environment variables.
+For example `OTEL_TRACES_EXPORTER=jaeger` configures your application to use the Jaeger exporter.
+The corresponding Jaeger exporter library has to be provided in the classpath of the application as well.
+
+It's also possible to set up the propagators via the `OTEL_PROPAGATORS` environment variable, 
+like for example using the `tracecontext` value to use [W3C Trace Context](https://www.w3.org/TR/trace-context/).
+
+For more details, see all the supported configuration options in the module's
+[README](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure).
```

Comment:
What's the difference between that file and https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/ ?

Reply:
What I see on your link it's agent related documentation. Of course the agent is configured through the SDK autoconfiguration but they are not related in the other way (you can use SDK autoconfiguration without the agent).
The first part of that link anyway refers to the README I put here.
I would leave the agent out imho.